autoflake.py

```diff
@@ -331,7 +331,8 @@ def filter_code(source, additional_imports=None,
                 expand_star_imports=False,
                 remove_all_unused_imports=False,
                 remove_duplicate_keys=False,
-                remove_unused_variables=False):
+                remove_unused_variables=False,
+                ignore_init_module_imports=False):
```

Comment:
`):` could be placed to the next line also , and add trailing comma on this line

Reply:
My bad, I let that slip through - but fixed now!